lte/gateway/c/core/oai/tasks/amf/amf_app_handler.cpp

```diff
@@ -1548,4 +1549,53 @@ void amf_app_handle_initial_context_setup_rsp(
   // update UE state
   ue_context->mm_state = REGISTERED_CONNECTED;
 }
+using grpc::Status;
+int amf_app_handle_pdu_session_failure(
+    itti_n11_create_pdu_session_failure_t* pdu_session_failure) {
+  if (!pdu_session_failure) {
+    return 0;
+  }
+  ue_m5gmm_context_s* ue_context = nullptr;
+
+  imsi64_t imsi64 = 0;
+  IMSI_STRING_TO_IMSI64(pdu_session_failure->imsi, &imsi64);
+  ue_context = lookup_ue_ctxt_by_imsi(imsi64);
+
+  if (!ue_context) {
+    OAILOG_ERROR(LOG_AMF_APP, "UE context is null\n");
+    return -1;
```

Comment:
Please use macros for return types.
```suggestion
    return RETURNerror;
```

Reply:
comment addressed